specs/windows/apps.table

```diff
@@ -0,0 +1,16 @@
+table_name("apps")
```

Comment:
This overloads the OS X `apps` table: https://osquery.io/docs/tables/#apps that is the source of the code audit and OS X/Linux build failures.

We can work through the differences and merge the two specs, or rename this table to `programs` (or something similar).